vm-setup/roles/v1aX_integration_test/tasks/node_reuse.yml

```diff
@@ -175,11 +175,10 @@
     delay: 2
     until:
       - bmhs is succeeded
-      - (saved_bmhs.resources | filter_provisioning("deprovisioning") | map(attribute='metadata.name')) |
-        intersect(bmhs.resources | filter_provisioning("available,ready") | map(attribute='metadata.name')) |
-        length ==  (NUMBER_OF_BMH_MD | int)
```

Comment:
why are we omitting `length` check btw?

Reply:
You are right, I overlooked the previous step, so we will always have only and one BMH in  `saved_bmhs` from the previous step which will be deprovisioning and becoming ready and registered in that var, and picking a first node from that ends up being the same BMH again, alright, fine with the logic above.